parsec/core/gui/central_widget.py

```diff
@@ -194,10 +198,17 @@ def _on_connection_state_changed(self, status, status_exc):
             tooltip = _("TEXT_BACKEND_STATE_CRASHED_cause").format(cause=str(status_exc.__cause__))
             icon = QPixmap(":/icons/images/material/cloud_off.svg")
             notif = ("ERROR", tooltip)
+            disconnected = True
 
         self.menu.set_connection_state(text, tooltip, icon)
         if notif:
             self.new_notification.emit(*notif)
+        if disconnected:
+            self.systray_notification.emit(
+                "Parsec",
+                f"{self.core.device.organization_id}:" + _("TEXT_BACKEND_STATE_DISCONNECTED"),
```

Comment:
The formatting here is done mostly in the code. I would rather we let the translator do all the formatting (as much as possible) with something like:
```python
_("TEXT_SYSTRAY_BACKEND_DISCONNECT_organization").format(organization=self.core.device.organization_id)
```
which allows us to use `{organization}` in the translation entry in the PO file.

Reply:
It's better, i defined it like that in PO:
```python
 msgid "TEXT_SYSTRAY_BACKEND_DISCONNECT_organization"                                                                                                                      
 msgstr "{organization}: Disconnected" 
```